src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,186 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> prev;
+        private Node<T> next;
+
+        Node(T item) {
+            this.item = item;
+        }
+
+        Node(T item, Node<T> prev) {
+            this.item = item;
+            this.prev = prev;
+        }
```

Comment:
It's better not to have this constructor. Because you have 2 possible places for this one argument: "prev" and "next". And it's not clear from code: which place it will go. Person, who will read your code, must always remember about this constructor. Your code must be as easy to read as possible. 
```suggestion
```